packages/storybook-vue/stories/3_components/toggle-group/ToggleGroup.stories.mdx

```diff
@@ -159,13 +163,83 @@ export const TemplateIcons = (args, { argTypes }) => ({
 </scale-toggle-group>
 ```
 
+### Scoped CSS variables
```

Comment:
I think we don't need to add scoped variables if they were not already there & accessible by the user. Only update the ones that have changed from before. Pls correct me if i'm wrong @acstll

Reply:
I was also not completely sure about this.